pkg/txhandler/registry/registry.go

```diff
@@ -45,14 +45,7 @@ type Factory interface {
 func RegisterHandler(factory Factory) string {
 	name := factory.Name()
 	txHandlers[name] = factory
-
-	// check whether a policy engine name is provided
-	if config.GetString(tmconfig.DeprecatedPolicyEngineName) != "" {
-		factory.InitConfig(tmconfig.DeprecatedPolicyEngineBaseConfig.SubSection(name))
-	} else {
-		// if not, use the new transaction handler configurations
-		factory.InitConfig(tmconfig.TransactionHandlerBaseConfig.SubSection(name))
-
-	}
+	// init the new transaction handler configurations
+	factory.InitConfig(tmconfig.TransactionHandlerBaseConfig.SubSection(name))
```

Comment:
Feels slightly odd that one subsection is passed in as an arg, and the other subsection is hard-coded inside the InitConfig method. Maybe both should be passed in _or_ both should be hardcoded?